modules/porous_flow/src/fvkernels/FVPorousFlowHeatAdvection.C

```diff
@@ -0,0 +1,103 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "FVPorousFlowHeatAdvection.h"
+#include "PorousFlowDictator.h"
+
+registerADMooseObject("PorousFlowApp", FVPorousFlowHeatAdvection);
+
+InputParameters
+FVPorousFlowHeatAdvection::validParams()
+{
+  InputParameters params = FVFluxKernel::validParams();
+  RealVectorValue g(0, 0, -9.81);
+  params.addParam<RealVectorValue>("gravity", g, "Gravity vector. Defaults to (0, 0, -9.81)");
```

Comment:
are there 2D porous flow simulations? This isnt the best default for 2D

Reply:
Yeah, also 1D. I've often thought that it should either default to (0, 0, 0) or have no default and be required. This default in the z direction is consistent with the existing kernels. 

I might take a poke around other modules and see what they do and change it to be consistent with them. I'll make an issue to fix these kernels (new and existing) if that is ok